### ISSUE TITLE
Sometimes travis aufs build may fail

### ISSUE THREAD

**Issue by gnawux** [hyperhq/hyperd#641]
```
W0607 02:34:04.890002   22371 aufs.go:177] Couldn't run auplink command : exit status 22
%!s(MISSING)
E0607 02:34:04.890222   22371 storage.go:291] got error when mount container to share dir Fail to mount aufs dir to /var/run/hyper/vm-bEqRGJZzON/share_dir/a627d41c3487871c6e267391076e84325fd7135352dbe51f7e13179024530b80/rootfs: invalid argument
E0607 02:34:04.890300   22371 container.go:887] Pod[pod-hWcGUSHwoQ] Con[990ed9fdbfa4(pod-hWcGUSHwoQ-busybox-0)] failed to prepare rootfs: Fail to mount aufs dir to /var/run/hyper/vm-bEqRGJZzON/share_dir/a627d41c3487871c6e267391076e84325fd7135352dbe51f7e13179024530b80/rootfs: invalid argument
```

full logs here: [log-aufs.txt](https://github.com/hyperhq/hyperd/files/1056776/log-aufs.txt)

**Comment by bergwolf** [comment]
There is an issue with aufs mount option length calculation. I've submitted https://github.com/hyperhq/hyperd/pull/642 to fix it. However, I am not sure if it is the root cause for the failure. I added some error logs so we can find out when we see the error again.

**Comment by gnawux** [comment]
closed by #642
